docs/dedicated-data-endpoints/dedicated-data-endpoints.md

```diff
@@ -0,0 +1,132 @@
+# Azure Container Registry: Dedicated Data Endpoints – Mitigating Data Exfiltration
+
+Azure Container Registry announces dedicated data-endpoints, enabling tightly scoped client firewall rules to specific registries, minimizing data exfiltration concerns.
+
+Pulling content from a registry involves two endpionts:
+
+- **Registry endpoint**, often refereed as the *login url*, used for authentication and content discovery.  
+  A command like `docker pull contoso.azurecr.io/hello-world` makes a REST request which authenticates and negotiates the layers which represent the requested artifact.
+- **Data-endpoints** serve blobs representing content layers.
+
+<img src="./media/registry-dual-endpoints.png" width=500x>
+
+## Registry Managed Storage Accounts
+
+Azure Container Registry is a multi-tenant service, where the data-endpoint storage accounts are managed by the registry service. There are many benefits for managed storage, such as load balancing, contentious content splitting, multiple copies for higher concurrent content delivery, and multi-region support with [geo-replication](https://aka.ms/acr/geo-replicatin).
+
+## Azure Private Link VNet Support
+
+Azure Container Registry recently announced [Private Link support](https://aka.ms/acr/privatelink), enabling a private endpoints from Azure VNets to be placed on the managed registry service. In this case, both the registry and data-endpoints are accessible from within the VNet, using private IPs. 
+
+The public endpoint can then be removed, securing the managed registry and storage accounts to access from within the VNet.
+
+<img src="./media/registry-private-link.png" width=500x>
+
+Unfortunately, VNet connectivity isn’t always an option.
+
+## Client Firewall Rules & Data Exfiltration Risks
+
+When connecting to a registry from on-prem hosts, IoT devices, custom build agents, or when Private Link may not be an option, client firewall rules may be applied, limiting access to specific resources.
+
+<img src="./media/registry-client-rules-all-storage.png" width=500x>
+
+As customers locked down their client firewall configurations, they realized they must create a rule with a wildcard for all storage accounts, raising concerns for data-exfiltration. A bad actor could deploy code that would be capable of writing to another "Evil-Empire" storage account.
+
```

Comment:
Not sure about term "Evil-Empire" here - didn't that refer to MSFT :). I see it in your diagram, perhaps substitute something like "Bad Actor"